src/NuGet.Clients/PackageManagement.VisualStudio/ScriptPackageFile.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Runtime.Versioning;
+using NuGet.Frameworks;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public class ScriptPackageFile : IScriptPackageFile
+    {
+        private readonly NuGetFramework _targetFramework;
+
+        public ScriptPackageFile(string path, NuGetFramework targetFramework)
+        {
+            Path = path.Replace(System.IO.Path.AltDirectorySeparatorChar, System.IO.Path.DirectorySeparatorChar);
```

Comment:
Path is a property in the interface IScriptPackageFile. So can't remove System.IO